RecoTracker/TkSeedingLayers/interface/SeedingHitSet.h

```diff
@@ -19,14 +19,15 @@ class SeedingHitSet {
 
   static ConstRecHitPointer nullPtr() { return nullptr; }
 
-  SeedingHitSet() { initContainer(); }
+  SeedingHitSet() { theSize = 0; }
```

Comment:
```suggestion
  SeedingHitSet() : theSize(0) {}
```
or even ` = default;` and have `unsigned int theSize = 0;` in the declaration.